### PR TITLE
[ISSUE-120] [BUG] Master‘s metrics of WorkerSlotsCount / WorkerSlotsUsed/ OverloadWorkerCount not update

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

**Comment by AngersZhuuuu** [comment]
@waitinfuture @FMX

File: server-master/src/main/scala/com/aliyun/emr/rss/service/deploy/master/Master.scala [hunk 1]
```diff
@@ -79,16 +79,21 @@ private[deploy] class Master(
...
       _ => statusSystem.blacklist.size())
+    val clusterSlotsUsageLimit: Double = RssConf.clusterSlotsUsageLimitPercent(conf)
     // worker count
     source.addGauge(MasterSource.WorkerCount,
       _ => statusSystem.workers.size())
-    val (totalSlots, usedSlots, overloadWorkerCount) = getClusterLoad
     // worker slots count
-    source.addGauge(MasterSource.WorkerSlotsCount, _ => totalSlots)
+    source.addGauge(MasterSource.WorkerSlotsCount,
+      _ => statusSystem.workers.asScala.map(_.numSlots).sum)
```
Comment (FMX) [inline server-master/src/main/scala/com/aliyun/emr/rss/service/deploy/master/Master.scala:88]: Variable "statusSystem.workers" includes workers on blackslist. I think using "workersSnapShot" will be better.

Comment (AngersZhuuuu) [reply server-master/src/main/scala/com/aliyun/emr/rss/service/deploy/master/Master.scala]: > Variable "statusSystem.workers" includes workers on blackslist. I think using "workersSnapShot" will be better.

Done

**Comment by FMX** [comment]
LGTM. Thanks.

**Pull request merged**